tests/compliance_operator_v2_test.go

```diff
@@ -312,7 +364,7 @@ func TestComplianceV2DeleteComplianceScanConfigurations(t *testing.T) {
 		Clusters: []string{clusterID},
 		ScanConfig: &v2.BaseComplianceScanConfigurationSettings{
 			OneTimeScan: false,
-			Profiles:    []string{"rhcos4-e8"},
+			Profiles:    []string{"rhcos4-high"},
```

Comment:
How come we're changing the profile? Did the e8 profile not clean up correctly in a previous test?

Reply:
that seems like the case, I am not sure what should I do here, should we wait for a clean up here or how do we trigger a clean up